.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "wcaForGP.enable": true
+}
```

Comment:
Is this meant to be included in the commit?

Reply:
I am not sure, Peter. This seems to have changed after the WCA extension was added to VS code as a part of the WCA@IBM initiative. If this doesn't have an impact on the OpenJ9 builds, then I guess it can be included in the commit.

Reply:
Pls remove it from the commit. You can update the https://github.com/eclipse-openj9/openj9-docs/blob/master/.gitignore file to ignore it in the future.

Reply:
ok